src/main/java/at/ac/tuwien/kr/alpha/grounder/structure/AnalyzeUnjustified.java

```diff
@@ -197,8 +217,7 @@ private ReturnExplainUnjust explainUnjust(LitSet x, Assignment currentAssignment
 
 			log("Checking atoms over predicate: {}", b.getPredicate());
 			AssignedAtomsIterator assignedAtomsOverPredicate = getAssignedAtomsOverPredicate(b.getPredicate());
-			atomLoop:
-			while (assignedAtomsOverPredicate.hasNext()) {
+			atomLoop: while (assignedAtomsOverPredicate.hasNext()) {
```

Comment:
Please revert the formatting.

Reply:
done

Reply:
This is similar to the above, unnecessary space and wrong indentation of the `for`.

Reply:
should be fixed now